rls/src/main/java/io/grpc/rls/RlsLoadBalancer.java

```diff
@@ -50,7 +50,6 @@ final class RlsLoadBalancer extends LoadBalancer {
 
   @Override
   public Status acceptResolvedAddresses(ResolvedAddresses resolvedAddresses) {
-    logger.log(ChannelLogLevel.DEBUG, "Received resolution result: {0}", resolvedAddresses);
```

Comment:
Logging the resolved addresses could be useful if the RLS server is returning something slightly broken.

Reply:
The only thing used by RLS in there is the LB policy config. I had kept the log where it said the config changed, but I see it isn't logging the config itself. I'll add logging the config there, thus it will be logged when meaningful.